src/stock.rs

```diff
@@ -356,22 +341,18 @@ mod tests {
                 DC_CONTACT_ID_SELF as u32
             ),
             "Member Alice (alice@example.com) added by me."
-        )
+        );
     }
 
     #[test]
     fn test_stock_system_msg_add_member_by_other_with_displayname() {
         let t = dummy_context();
-        let contact_id = unsafe {
-            let name = CString::new("Alice").unwrap();
-            let addr = CString::new("alice@example.com").unwrap();
+        let contact_id = {
             assert!(
-                dc_create_contact(&t.ctx, name.as_ptr(), addr.as_ptr()) > 0,
+                Contact::create(&t.ctx, "Alice", "alice@example.com").unwrap() > 0,
```

Comment:
Doesn't `Contact::create()` return `Result` now?  Should this not become `.expect("Failed to create contact Alice")` instead of just `.unwrap()`.  Or keep `.unwrap()` but just get rid of the `assert!()`, I think it was there only make sure the contact creation was successful which `.uwrap()` (or `.expect()`) would do much better.